### PR TITLE
meta2: Fix versioning + Miscellaneous fixes

### PR DESCRIPTION
**Pull request by jfsmig**
* meta2: fix the parsing of version numbers (now accepting int64 integers)
* Tune some variables in the central configuration, for starvation less reactive
* Instead, let's print a warning when there is an early clue of a starvation (new variable added)

### CODE REVIEW

File: conf.json [hunk 1]
```diff
@@ -354,13 +354,18 @@
 
 			{ "type": "monotonic", "name": "server_queue_max_delay",
 				"key": "server.queue.max_delay",
-				"descr": "Anti-DDoS counter-mesure. In the current server, sets the maximum amount of time a queued TCP event may remain in the queue. If an event is polled and the thread sees the event stayed longer than that delay, the connection is immediately closed.",
-				"def": "1s", "min": "1ms", "max": "1h" },
+				"descr": "Anti-DDoS counter-mesure. In the current server, sets the maximum amount of time a queued TCP event may remain in the queue. If an event is polled and the thread sees the event stayed longer than that delay, the connection is immediately closed. Keep this value rather high because the connection closing doesn't involve a reply that will help the client to retry with an exponential back-off.",
+				"def": "5s", "min": "10ms", "max": "1h" },
+
+			{ "type": "monotonic", "name": "server_queue_warn_delay",
+				"key": "server.queue.max_delay",
```
Comment (fvennetier) [inline conf.json]: I think you wanted to write `server.queue.warn_delay`.

File: proxy/m2_actions.c [hunk 2]
```diff
@@ -1679,11 +1679,11 @@ static enum http_rc_e action_m2_content_touch (struct req_args_s *args,
...
-	if ((!oio_url_has_fq_container(args->url) &&
-		 !oio_url_has(args->url, OIOURL_HEXID)) ||
-			(!oio_url_has(args->url, OIOURL_PATH) &&
-			 !oio_url_has(args->url, OIOURL_CONTENTID)))
-		return _reply_format_error(args, BADREQ("Missing content path or ID"));
+	if (!oio_url_has_fq_container(args->url))
+		return _reply_format_error(args, BADREQ("container unspecified"));
+	if (!oio_url_has(args->url, OIOURL_PATH) &&
+			!oio_url_has(args->url, OIOURL_CONTENTID))
+		return _reply_format_error(args, BADREQ("content unspecified"));
```
Comment (fvennetier) [inline proxy/m2_actions.c]: "Missing content path or ID" was clearer than "content unspecified".

File: Variables.CMakeFile [hunk 3]
```diff
@@ -72,6 +73,7 @@ dir2macro(OIO__SERVER_POOL_MAX_TCP)
 dir2macro(OIO__SERVER_POOL_MAX_UDP)
 dir2macro(OIO__SERVER_POOL_MAX_UNUSED)
 dir2macro(OIO__SERVER_QUEUE_MAX_DELAY)
+dir2macro(OIO__SERVER_QUEUE_MAX_DELAY)
```
Comment (fvennetier) [inline Variables.CMakeFile]: Twice the same macro?

Comment (jfsmig) [inline Variables.CMakeFile]: This file is auto-generated, the error is a consequence of your previous message.

File: proxy/m2_actions.c (hunk 2 above)
Comment (fvennetier) [inline proxy/m2_actions.c]: Now there is a typo in the correction...

**Pull request merged**